polyphemus/Makefile

```diff
@@ -7,6 +7,12 @@ include ../make-base/stubs.mk
 	else \
 		touch .projects-mark; \
 	fi
+	git clone git@github.com:mountetna/renaming-projects.git lib/etls/renaming/projects || CLONE_FAILED=1
```

Comment:
I think you want this to happen before .projects-mark is touched, otherwise this might only attempt once, and on failure, not attempt again.

See how the `docker-ready` command depends on `.projects-mark` to run this target.  The touching of the file is what marks this as "done successfully"